azavea/cac-tripplanner
# Hide drag handles on event place cards

## Overview

Hide drag handles added in #1168 from event place cards, which are not routable or draggable.


## Testing Instructions

 * Events map sidebar place cards should not show drag handle on hover (or be draggable)
 * Tours map sidebar place cards should still show drag handles on hover (and be draggable)


Fixes #1178.
